templates/_schemas/definitions/listItem.twig

```diff
@@ -2,8 +2,9 @@
   
   {% set schema = schema ?? craft.schema.ListItem %}
   {% set schema = schema
-    .position(props.listPosition)
-    .url(props.url)
+    .position(props.listPosition|default(null))
+    .name(props.title|default(null))
```

Comment:
Both `name` and `url` properties should be auto-set by the `Thing` macro.

Reply:
Done.